CHANGES.rst

```diff
@@ -1,6 +1,15 @@
 Release notes
 =============
 
+v1.0.10 2025-06-25
+------------------
+* Reorganisation of project structure for wheel support
+* Round to nearest integer for star polymer arms
```

Comment:
Not seeing star polymer changes in the changeset. Was it previously applied without updating release notes?